CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+x.x.x Release notes (yyyy-mm-dd)
+=============================================================
+### Enhancements
+* None.
+
+### Fixed
+* Tokens are refreshed ahead of time. If the lifetime of the token is lower than the threshold for refreshing it will cause the client to continously refresh, spamming the server with refresh requests. A lower bound of 10 seconds have been introduced. ([#2115](https://github.com/realm/realm-js/issues/2115), since 1.0.2)
```

Comment:
```suggestion
* Tokens are refreshed ahead of time. If the lifetime of the token is lower than the threshold for refreshing it will cause the client to continously refresh, spamming the server with refresh requests. A lower bound of 10 seconds has been introduced. ([#2115](https://github.com/realm/realm-js/issues/2115), since v1.0.2)
```